db-commons/src/main/scala/org/bitcoins/db/DbCommonsColumnMappers.scala

```diff
@@ -166,6 +183,83 @@ abstract class DbCommonsColumnMappers {
     MappedColumnType
       .base[TxoState, String](_.toString, TxoState.fromString(_).get)
   }
+
+  implicit val oracleInfoMapper: BaseColumnType[OracleInfo] = {
+    MappedColumnType
+      .base[OracleInfo, String](_.hex, OracleInfo.fromHex)
+  }
+
+  implicit val contractInfoMapper: BaseColumnType[ContractInfo] = {
```

Comment:
Hmm... it doesn't seem ideal to store this in it's serialized version as then it makes it hard to query on the database. If i'm understanding this correctly I couldn't run queries on different contract types because it's just a binary blob at the database level. This comment applies with other mappers as well, I think we want to try and figure out a way to store them so that it is easy to run traditional sql queries on.